zerver/webhooks/newrelic/doc.md

```diff
@@ -1,27 +1,70 @@
+# New Relic ClickUp integration
+
 New Relic can send messages to a Zulip channel for incidents.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. On [New Relic](https://one.newrelic.com),
-  select **Alerts & AI**.
+1. Go to the [**Destinations**](https://one.newrelic.com/alerts-ai/notify/destinations) menu and select **Webhook** in the **Add a destination** section. ([*Official Guide*](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#set-webhook-destination))
```

Comment:
This whole file needs to be updated for our line break conventions for documentation files. Generally, a max line length of 76 char is fine.

Reply:
Also, I'd put the links at the bottom of the file and use reference links. It improves readability.